src/middleware/ErrorHandler.ts

```diff
@@ -0,0 +1,8 @@
+import { Request, Response, NextFunction } from 'express';
+
+export function handler(err: any, req: Request, res: Response, nextFunction: NextFunction): void {
+    console.error(err.message);
+    if (!err.statusCode) err.statusCode = 500;
```

Comment:
we could use a library for the status codes https://www.npmjs.com/package/http-status-codes

Reply:
I like that 👍